File: Python Word examples/field/addMergeField.py
import spirecloudword
from spirecloudword.configuration import Configuration as WordConfiguration

appId = "your id"
appKey = "your key"

configuration = WordConfiguration(appId, appKey)
api = spirecloudword.api.fields_api.FieldsApi(configuration)

file_name = "Template.docx"
paragraph_path = "sections/0/paragraphs/0"
field_name = 'FiledTest'
extra = {'folder': 'input',
         'index_in_paragraph': 0,
         'dest_file_path': 'output/AddMergeField.docx'}
api.add_merge_field(file_name, paragraph_path, field_name, **extra)

File: PHP Word Examples/FieldsApiDemo/addMergeField.php
<?php

use Spire\Cloud\Word\Sdk\Client\Api\FieldsApi;
use Spire\Cloud\Word\Sdk\Client\Configuration;

$appId = "your id";
$appKey = "your key";
$baseUrl="https://api.e-iceblue.cn";
$configuration = new Configuration($appId, $appKey,$baseUrl);
$apiInstance = new FieldsApi($configuration);

$name = "AddMergeField.docx";
$field_name = 'test_name';
$paragraph_path = 'Section/0/Body/0/Paragraph/0';
$folder = "input";
$storage = null;
$index_in_paragraph = 1;
$field_code = null;
$password = null;
$dest_file_path = "output/AddMergeField.docx";;
$apiInstance->addMergeField($name, $paragraph_path, $field_name, $dest_file_path, $folder, $storage, $password, $index_in_paragraph, $field_code);